perfdash/config.go

```diff
@@ -42,36 +45,85 @@ type Jobs map[string]Tests
 type Buckets map[string]Jobs
 
 var (
+	// performanceDescriptions contains metrics exported by a --ginko.focus=[Feature:Performance]
+	// e2e test
+	performanceDescriptions = map[string]TestDescription{
+		"DensityResponsiveness": {
+			Name:             "density",
+			OutputFilePrefix: "APIResponsiveness",
+			Parser:           parseResponsivenessData,
+		},
+		"DensityResources": {
+			Name:             "density",
+			OutputFilePrefix: "ResourceUsageSummary",
+			Parser:           parseResourceUsageData,
+		},
+		"DensityPodStartup": {
+			Name:             "density",
+			OutputFilePrefix: "PodStartupLatency",
+			Parser:           parseResponsivenessData,
+		},
+		"LoadResponsiveness": {
+			Name:             "load",
+			OutputFilePrefix: "APIResponsiveness",
+			Parser:           parseResponsivenessData,
+		},
+		"LoadResources": {
+			Name:             "load",
+			OutputFilePrefix: "ResourceUsageSummary",
+			Parser:           parseResourceUsageData,
+		},
+	}
+
 	// TestConfig contains all the test PerfDash supports now. Downloader will download and
 	// analyze build log from all these Jobs, and parse the data from all these Test.
 	// Notice that all the tests should have different name for now.
 	TestConfig = Buckets{
 		utils.KubekinsBucket: Jobs{
 			"ci-kubernetes-e2e-gci-gce-scalability": Tests{
-				"DensityResponsiveness": TestDescription{
-					Name:             "density",
-					OutputFilePrefix: "APIResponsiveness",
-					Parser:           parseResponsivenessData,
-				},
-				"LoadResponsiveness": TestDescription{
-					Name:             "load",
-					OutputFilePrefix: "APIResponsiveness",
-					Parser:           parseResponsivenessData,
-				},
-				"DensityResources": TestDescription{
-					Name:             "density",
-					OutputFilePrefix: "ResourceUsageSummary",
-					Parser:           parseResourceUsageData,
-				},
-				"LoadResources": TestDescription{
-					Name:             "load",
-					OutputFilePrefix: "ResourceUsageSummary",
-					Parser:           parseResourceUsageData,
-				},
+				Prefix:       "gce-100Nodes-master",
+				Descriptions: performanceDescriptions,
+			},
+			"ci-kubernetes-e2e-gci-gce-scalability-stable1": Tests{
```

Comment:
Having names of our CI jobs hard-coded here doesn't seem too clean. We're changing it from time and time and every time we do that, we'll have to change here also. Could you we do something better? Maybe just get the names of the jobs from [config.json](https://github.com/kubernetes/test-infra/blob/master/jobs/config.json) and select ones owned by sig-scale?

Reply:
But how to retrieve the contents of this file? It's in a different repo, with its own change cadence.

Reply:
Maybe just `wget https://github.com/kubernetes/test-infra/blob/master/jobs/config.json`?

Reply:
FTR, opened https://github.com/kubernetes/contrib/issues/2812 about this.